cdap-docs/integrations/source/mapr/index.rst

```diff
@@ -0,0 +1,51 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _mapr-configuring:
+
+=======================================
+Configuring and Installing CDAP on MapR
+=======================================
+
+.. highlight:: console
+
+This section will assist you with installing CDAP on `MapR <https://www.mapr.com>`__. CDAP
+can be installed on a MapR cluster by following the :ref:`manual instructions for installing
+CDAP <install>`. These notes are an addition to those steps.
+
+Note that as described in the :ref:`Software Prerequisites <install-software-requirements>`, 
+a configured Hadoop and HBase (plus an optional Hive client) needs to be configured
+on the node(s) where CDAP will run.
+
+- To configure a MapR Hadoop client, see the MapR documentation on `Setting Up the Client
```

Comment:
They tend to call it just the MapR client, not a Hadoop client.  (Technically it is a client to their hdfs-alternative and a Yarn client, so i can see both sides)